pom.xml

```diff
@@ -331,27 +337,96 @@
                     <goals>deploy</goals>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
It was duplicated right above it (https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/83/files#diff-600376dffeb79835ede4a0b285078036R318)